File: pset6/mario.py
from cs50 import get_int

height = 0

while height <= 0 or height > 8:
    height = get_int("Height: ")

for i in range(height):
    j = height - i
    while j > 1:
        print(" ", end="")
        j -= 1
    for hash in range(i + 1):
        print("#", end="")
        hash += 1
    print()
    i += 1

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height<=0 || height>8);

    for (int l = 0; l < height; l++)
    {
        for (int s = height-l; s > 1; s--)
        {
            printf(" ");
        }
        for (int hash = 0; hash < l+1; hash++)
        {
            printf("#");
        }
        printf ("\n");
    }
}
